plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -50,5 +59,88 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 			</div>
 		);
 	}
-	return <div className="woocommerce-attribute-field"></div>;
+
+	const sortedAttributes = value.sort( ( a, b ) => a.position - b.position );
+	const attributeKeyValues = value.reduce(
+		(
+			keyValue: Record< number, ProductAttribute >,
+			attribute: ProductAttribute
+		) => {
+			keyValue[ attribute.id ] = attribute;
+			return keyValue;
+		},
+		{} as Record< number, ProductAttribute >
+	);
+	return (
+		<div className="woocommerce-attribute-field">
+			<Sortable
+				onOrderChange={ ( items ) => {
+					const newAttributes: ProductAttribute[] = items
+						.map( ( item, index ): ProductAttribute | undefined => {
```

Comment:
I think this callback function could really useful as a named and exported method in a utils file.  This would have a few key benefits:

* We could name it to more quickly identify what it's doing
* We'll be able to test the method
* We'll minimize the amount of logic directly inside this component